drivers/docker/docklog/docker_logger_test.go

```diff
@@ -18,9 +18,9 @@ import (
 
 func testContainerDetails() (image string, imageName string, imageTag string) {
 	if runtime.GOOS == "windows" {
-		return "dantoml/busybox-windows:08012019",
-			"dantoml/busybox-windows",
-			"08012019"
+		return "stefanscherer/busybox-windows:latest",
```

Comment:
We should use a specific tag rather than `latest` here, so as to reduce the risk of unexpected changes.